package.json

```diff
@@ -37,19 +37,23 @@
     "css-loader": "~0.23.1",
     "del": "*",
     "edx-ui-toolkit": "~1.4.1",
+    "eslint": "~2.13.1",
+    "eslint-config-edx": "~1.1.0",
     "extract-text-webpack-plugin": "~1.0.1",
     "file-loader": "0.8.5",
     "font-awesome-loader": "0.0.1",
     "gulp": "*",
     "gulp-changed": "*",
     "gulp-concat": "*",
+    "gulp-eslint": "^2.0.0",
```

Comment:
So, I had issues in the ui-toolkit with `gulp-eslint` taking an _increeeeedibly_ long time to run (like 30sec) - did you run into that here? I fixed it by using `gulp-shell` and having it just run `eslint . ` in bash behind the scenes (which takes like 3sec).